Cargo.toml

```diff
@@ -15,7 +15,7 @@ default = []
 driver_mongodb = ["mongodb", "bson"]
 
 [dependencies]
-bson = { version = "0.2.0", optional = true }
+bson = { version = "0.1.4", optional = true }
```

Comment:
```
<bson macros>:6:1: 6:9 error: mismatched types:
 expected `bson::ordered::OrderedDocument`,
    found `bson::ordered::OrderedDocument`
```

The MongoDB crate uses `0.1.4` so in order to avoid duplicates we downgrade.